### PR TITLE
Replace saveorsend strings

### PR DESCRIPTION
**Pull request by froberg**
Will do employer after this PR is completed. 
/c

### CODE REVIEW

File: src/SFA.DAS.ProviderApprenticeshipsService.Web.Tests/Orchestrators/WhenApproveCommitment.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+    using SFA.DAS.Commitments.Api.Types;
+
+    using Web.Orchestrators;
+
+    [TestFixture]
+    public class WhenApproveCommitment
+    {
+        [TestCase(SaveStatus.Save, AgreementStatus.NotAgreed, false)]
+        public async Task  Bla(SaveStatus input, AgreementStatus test1, bool test2)
```
Comment (leewadhams) [inline src/SFA.DAS.ProviderApprenticeshipsService.Web.Tests/Orchestrators/WhenApproveCommitment.cs]: Name :-)

File: src/SFA.DAS.ProviderApprenticeshipsService.Application/Commands/SubmitCommitment/SubmitCommitmentCommand.cs [hunk 2]
```diff
@@ -10,6 +12,8 @@ public class SubmitCommitmentCommand : IAsyncRequest
 
         public string Message { get; set; }
 
-        public string SaveOrSend { get; set; }
+        public AgreementStatus AgreementStatus { get; set; }
```
Comment (leewadhams) [inline src/SFA.DAS.ProviderApprenticeshipsService.Application/Commands/SubmitCommitment/SubmitCommitmentCommand.cs]: A couple of questions here. Should we be passing ApiTypes to the Command Handler + Should the Command handler decide whether or not to create a task? Thoughts @mgwilliam ?

File: src/SFA.DAS.ProviderApprenticeshipsService.Web.Tests/Orchestrators/WhenApproveCommitment.cs (hunk 1 above)
Comment (froberg) [inline src/SFA.DAS.ProviderApprenticeshipsService.Web.Tests/Orchestrators/WhenApproveCommitment.cs]: Not again....

**Comment by froberg** [comment]
@leewadhams -> Name / test is updated

File: src/SFA.DAS.ProviderApprenticeshipsService.Application/Commands/SubmitCommitment/SubmitCommitmentCommand.cs (hunk 2 above)
Comment (mgwilliam) [inline src/SFA.DAS.ProviderApprenticeshipsService.Application/Commands/SubmitCommitment/SubmitCommitmentCommand.cs]: i don't think we should BUT all of the command handlers currently do this. instead the orchestrator should map to domain entities (ie. the commands should know about entities not api types)

i think this is a refactor we need to do at some point across all the commands as a separate ticket

Comment (mgwilliam) [inline src/SFA.DAS.ProviderApprenticeshipsService.Application/Commands/SubmitCommitment/SubmitCommitmentCommand.cs]: @leewadhams, @froberg, can we discuss this morning about where to create the task

**Pull request merged**